web/v1/bigquery/sql-runner/playbooks/standard/00-setup/00-setup-metadata.yml.tmpl

```diff
@@ -5,8 +5,8 @@
   :region:
 :variables:
   :model_version:      bigquery/web/1.0.0
-  :scratch_schema:     scratch
-  :output_schema:      derived
+  :scratch_schema:     scratch_prod1
```

Comment:
Out of interest why do we use this prod suffix for BQ but not RS? Otherwise all looks good to me.

Reply:
`scratch`, `derived`, `atomic` are the historic defaults - because that's what we defaulted to deploy with.

Now we default to append the pipeline name `_prod1` to them all.

They should be consistent across all - the idea in going with `_prod1` was to change all to have `_prod_1`. But I keep flip-flopping in my head about whether this is a good choice. I'll pop the Q into the team slack - it's annoying me at this point!